internal/config/lb_test.go

```diff
@@ -53,53 +55,122 @@ func TestLB_Bind(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           AgentPort: 0,
-		           AgentName: "",
-		           AgentNamespace: "",
-		           AgentDNS: "",
-		           NodeName: "",
-		           IndexReplica: 0,
-		           Discoverer: DiscovererClient{},
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           AgentPort: 0,
-		           AgentName: "",
-		           AgentNamespace: "",
-		           AgentDNS: "",
-		           NodeName: "",
-		           IndexReplica: 0,
-		           Discoverer: DiscovererClient{},
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		func() test {
+			return test{
+				name: "return LB when the bind successes and the Discoverer is nil",
+				fields: fields{
+					AgentPort:      8081,
+					AgentName:      "vald-agent-ngt",
+					AgentNamespace: "vald",
+					AgentDNS:       "vald-agent-ngt.vald.svc.cluster.local",
+					NodeName:       "vald-01-worker",
+					IndexReplica:   3,
+				},
+				want: want{
+					want: &LB{
+						AgentPort:      8081,
+						AgentName:      "vald-agent-ngt",
+						AgentNamespace: "vald",
+						AgentDNS:       "vald-agent-ngt.vald.svc.cluster.local",
+						NodeName:       "vald-01-worker",
+						IndexReplica:   3,
+					},
+				},
+			}
+		}(),
```

Comment:
How about define the specific variable before `return test{}` and use them in `test{}` struct?
like as:
```go
func () test {
    agentPort := 8081
    return test {
        ...
        fields: fields{
            AgentPort: agentPort,
            ....
        },
        want: want{
            want: &LB {
                AgentPort: agentPort,
                ....
            },
            ....
        }
    }
}
```
You use the functional test codes, you can use them for more readable and easy to maintenance.

Reply:
thank you for your review.
I think it looks good to me!
I will fix it soon!